### PR TITLE
[AF-1467] Upgrade to WildFly 13

### PR DESCRIPTION
**Pull request by ederign**
Depends on https://github.com/jboss-integration/jboss-integration-platform-bom/pull/421

### CODE REVIEW

File: errai-bom/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
   <parent>
     <groupId>org.jboss.integration-platform</groupId>
     <artifactId>jboss-integration-platform-parent</artifactId>
-    <version>8.0.0.CR3</version>
+    <version>8.1.0-SNAPSHOT</version>
```
Comment (ederign) [inline errai-bom/pom.xml]: @mbiarnes @tiagobento it is my first wf upgrade.

How we make the build green without setting a snapshot version here?

**Comment by kiereleaseuser2** [comment]
Build finished. 2700 tests run, 5 skipped, 0 failed.

File: errai-bom/pom.xml (hunk 1 above)
Comment (tiagobento) [reply errai-bom/pom.xml]: I don't know. Ideally we shouldn't use SNAPSHOT versions.. There's a `8.1.0.CR1` artifact on `JBoss Releases`, are we able to use that?

Comment (tiagobento) [reply errai-bom/pom.xml]: Nevermind, just saw https://github.com/errai/errai/pull/356.

**Comment by ederign** [comment]
This will be replaced by a wf14 updgrade done by @adrielparedes

**Pull request closed without merging**